user/useradm.go

```diff
@@ -329,26 +334,50 @@ func (ua *UserAdm) compensateTenantUser(ctx context.Context, userId, tenantId st
 	if err != nil {
 		return errors.Wrap(err, "faield to delete tenant user")
 	}
+	return nil
+}
 
+func (ua *UserAdm) validateUserUpdate(
+	ctx context.Context,
+	me bool,
+	user *model.User,
+	u *model.UserUpdate,
```

Comment:
nitpick: I'd move the "me" argument in 4th position (ctx, user, u, me).
It looks more natural to my eyes.

Reply:
done